python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -214,6 +214,34 @@ def _checkProxy( delay=60, renew = True, shouldRaise = True, force = False ):
             last_modified_time = time.time()
 
 
+def getAccessURLs(lfns):
```

Comment:
Can we introduce some code quality tests on Jenkins to highlight these things? I've seen this come up before somewhere.